plugin/src/main/java/org/wildfly/plugin/server/RunMojo.java

```diff
@@ -360,7 +371,7 @@ private Path provisionIfRequired(final Path installDir) throws MojoFailureExcept
         try {
             if (!Files.exists(installDir)) {
                 getLog().info("Provisioning default WildFly server in " + installDir);
```

Comment:
Perhaps we could change this sentence to: "Provisioning default server in"

Reply:
+1 that makes sense.